Add readme files for Backup and Restore (#596)
kyma-project/infrastructure-manager: hack/runtime-migrator/README-rm.md

> 1. connects to a Gardener project

Reviewer: Applied in 759c9ac25eda6cb360352c09bfd9b4f7b47bfdf7

2. retrieves all existing shoot specifications
3. migrates the shoot specs to the new Runtime custom resource (Runtime CRs created with this migrator have the `operator.kyma-project.io/created-by-migrator=true` label)

> 4. saves the new Runtime custom resources to files

Reviewer 2: ```suggestion
4. saves the new Runtime CR to files
```

> 5. checks if the new Runtime custom resource will not cause update on the Gardener

Reviewer 2: ```suggestion
5. checks if the new Runtime CR won't cause an update on Gardener
```
